index.html

```diff
@@ -4,9 +4,44 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maple-ui.netlify.app/CSS/base.css">
+    <link rel="stylesheet" href="./CSS/quiz-home.css">
     <title>Document</title>
 </head>
 <body>
-    
+    <nav class="nav-bar">
+        <div class="nav-left">
+            <a class=" txt-xl semibold-font header-text-color" href="#">Maple Quiz</a>
+        </div>
+        <div class="nav-right">
+            <a href="" class=" txt-lg header-text-color">Docs</a>
+            <a href=""><i class="fab fa-github txt-xl header-text-color"></i></a> 
+        </div>
+    </nav>
+    <main>
+        <div class="main-wrap">
+            <div class="main-content-area">    
+                <div class="text-div">
+                    <div class="para-div">
+                        <p class="paragraph ">Let's have some fun playing quiz</p>
+                    </div>
+                    <div class="para-button">
+                        <a href="./quiz-cate.html" class="btn quiz-btn-background-color ">See Categories</a>
+                        <a href="./quiz-rules.html" class="btn quiz-btn-sec">Rules</a>
+                    </div>
+                </div>
+                <div class="hero-img-div">
+                    <img class="hero-img img-res shadow" src="./Assests/think-final.png" alt="">
```

Comment:
you should always mention **alt** attribute of image

Reply:
already updated in the above commit.